.babelrc

```diff
@@ -0,0 +1,5 @@
+{
+  "presets": [
+    "@babel/preset-env"
+  ]
+}
```

Comment:
@LukasChiama  Put a new line after this LOC... I expected ESLint (AirBnB) to flag it if there are no new lines at the end of any file

Reply:
Okay. ESLint does not check configuration files for errors and this is a configuration file.